arches_controlled_lists/models.py

```diff
@@ -219,6 +224,30 @@ def build_tile_value(self):
         }
         return tile_value
 
+    def build_select_option(self):
+        labels = getattr(self, "list_item_labels", self.list_item_values.labels())
+        ranked_labels = sorted(
+            labels,
+            key=lambda label: rank_label(
+                kind=label.valuetype_id,
+                source_lang=label.language_id,
+                target_lang=translation.get_language(),
+            ),
+            reverse=True,
+        )
+        data = {
+            "list_item_id": str(self.id),
+            "uri": self.uri,
+            "list_item_values": [label.serialize() for label in labels],
+            "display_label": ranked_labels[0].value,
+            "sortorder": self.sortorder,
+        }
+        data["children"] = sorted(
+            [child.build_select_option() for child in self.children.all()],
+            key=lambda d: d["sortorder"],
```

Comment:
We should probably just get this field declared as the default [ordering](https://docs.djangoproject.com/en/5.1/ref/models/options/#ordering) on the model.